server/controllers/createGroup.js

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import PostIt from './../src/PostIt';
+import verifyToken from './../middleware/verifyToken';
+
+const createGroup = express.Router();
+const postIt = new PostIt();
+
+/**
+ * Creates a new group
+ * Rejects request if the user is unauthenticated
+ * Refuses to create group if group name is empty
+ */
+createGroup.post('/groups', verifyToken, (request, response) => {
+  const { name, description } = request.body;
+
+  if (!name || !description) {
+    response.status(400).json({
```

Comment:
Try and handle more potential errors here, such as white-space. It's possible a name or description containing only white-space would pass through this endpoint, and i can't see any middleware you'v put in place

Reply:
A lot of the errors are handled in the middleware `verifyMembership.js`. Check it out

Reply:
check createGroup.post().....the only middleware there is verifyToken